handlers/bounty_test.go

```diff
@@ -1269,4 +1269,70 @@ func TestMakeBountyPayment(t *testing.T) {
 
 	})
 
+
 }
+
+func TestMakeBountyPayments(t *testing.T) {
+	pubKey := "test-pubkey"
+	ctx := context.WithValue(context.Background(), auth.ContextKey, pubKey)
+	mockDb := dbMocks.NewDatabase(t)
+	mockHttpClient := mocks.NewHttpClient(t)
+	bHandler := NewBountyHandler(mockHttpClient, mockDb)
+	jsonBody := `{"someKey": "someValue"}`
+	existingBounty := db.Bounty{
+		ID:          12,
+		Type:        "coding",
+		Title:       "first bounty",
+		Description: "first bounty description",
```

Comment:
@gouravmpk I can't see you mocking the relay calls

Reply:
@elraphty had a discussion with @ecurrencyhodler regarding this issue this test is already done by someone